doc/ARS/Local_ARS_HLD.md

```diff
@@ -0,0 +1,1151 @@
+            }
+
+            leaf primary_path_threshold {
+                type uint32;
+                default 16;
+                description  "Primary path metric";
+            }
+
+            leaf alternative_path_cost {
+                type uint32;
+                default 0;
+                description  "Alternative path cost";
+            }
+        }
+        /* end of list ARS_OBJECT_LIST */
+    }
+    /* end of container ARS_OBJECT */
+```
+
+##### ARS_NEXTHOPS
+
+```
+    container ARS_NEXTHOPS{
+
+        description "Nexthop IPs for identifying ARS-enabled NHG";
+
+        list ARS_NEXTHOP_LIST {
+
+            key "vrf_name nexthop_ip";
+
+            leaf vrf_name {
+                type union {
+                    type string {
+                        pattern "default";
+                    }
+                    type leafref {
+                        path "/vrf:sonic-vrf/vrf:VRF/vrf:VRF_LIST/vrf:name";
+                    }
+                }
+                description "VRF name";
+            } 
+
+            leaf nexthop_ip {
+                type inet:ip-address;
+                description "Nexthop-IP which is a member if nexthop group for which ARS behavior is desired";
+            }
+
+            leaf ars_obj_name {
+                description "ARS object name, inherited from 'default_ars_object' unless explicitly configured";
+                type leafref {
+                    path "/sars:sonic-ars/sars:ARS_OBJECT/sars:ARS_OBJECT_LIST/sars:obj_name";
+                }
+            }
+            leaf role {
+                type enumeration {
+                    enum primary_path{
+                        description "Member is participating in primary path";
+                    }
+                    enum alternative_path {
+                        description "Member is participating in alternative path";
+                    }
+                }
+                default primary_path;
+                description "NHG member's role";
+            }
+        }
+        /* end of list ARS_NEXTHOP_LIST */
+    }
+    /* end of container ARS_NEXTHOPS*/
+```
+##### ARS_INTERFACES
+
+```
+    container ARS_INTERFACES {
+
+        list ARS_INTERFACE_LIST {
+            description  "List of configured interfaces participating in ARS and
+                          used for ARS object selection when ARS_PROFILE's ars_nhg_path_selector_mode
+                          as interface/nexthop and also, used for ARS object selection
+                          when ARS_PROFILE's ars_lag_path_selector_mode as global/interface”
+            key "if_name";
+
+            leaf if_name {
+                type leafref {
+                    path "/port:sonic-port/port:PORT/port:PORT_LIST/port:name";
+                }
+                description "ARS-enabled interface name";
+            }
+
+            leaf scaling_factor {

```

Comment:
the definition of "scaling_factor" differs from SAI_PORT_ATTR_ARS_PORT_LOAD_SCALING_FACTOR and brings ambiguity. 

`scaling_factor`: 10000
`SAI_PORT_ATTR_ARS_PORT_LOAD_SCALING_FACTOR`:40 # for 400G

Why not just set the value of SAI_PORT_ATTR_ARS_PORT_LOAD_SCALING_FACTOR to be value of scaling_factor?

`scaling_factor`: 40
`SAI_PORT_ATTR_ARS_PORT_LOAD_SCALING_FACTOR`:40 # for 400G